adrs/0001-pseudo-private-custom-properties.md

```diff
@@ -0,0 +1,28 @@
+# Refactor component styling using pseudo-private custom properties
+
+**Date:** [2022-11-03]
+
+### Issue
+Component styles are difficult to maintain. This is due to:
+
+1. Disjointed styles .less files amongst components. Within components, we currently sometime will output contextual, variant, modifier, interaction styles with the base component class and sometimes we don’t. These styles will might be in alphabetical order, by some logical order, in the order added, or seemingly randomly. Many component style .less files include a large preamble and table of contents that are seldom updated to reflect the current state of the component. 
```

Comment:
> Disjointed styles .less files amongst components. Within components, we currently sometime will output contextual, variant, modifier, interaction styles with the base component class and sometimes we don’t.

How complicated would be to add a simple snippet example from one of our files? Sometimes few lines of code are easier to digest than words. 🙂

Reply:
Oh, I should have proofread this! It's tough to understand. I'll give this section an edit. A snippet might be tough, since the issue could be quite verbose, but I'll give it a shot.

Thanks for pointing this out!